src/commcare_cloud/fab/fabfile.py

```diff
@@ -286,7 +284,6 @@ def env_common():
         'pgstandby': pg_standby,
         'elasticsearch': elasticsearch,
         'riakcs': riakcs,
-        'rabbitmq': rabbitmq,
```

Comment:
why did this get removed?

Reply:
I'm removing it here from fab because it's unused, and because on the new swiss server we'll be using redis as the celery backend instead of rabbitmq. _That_ is in turn because of the reasons @emord noted here: https://github.com/dimagi/commcare-cloud/pull/2917.